src/menu/menu.less

```diff
@@ -70,3 +73,17 @@
         margin: 4px 0;
     }
 }
+
+.reactist_menulist [role='menuitem'] .reactist_menuitem_icon {
+    width: 24px;
```

Comment:
I wouldn't do it. This is not spacing conceptually, even if the value happens to be the same as the `xlarge` spacing. This is our standard icon size for menus. I agree it could be a design token encoded in a CSS variable, but I do not have the need or space right now to deal with standardizing it in such way yet.